tools/monitoring/config-getter/sample.yaml

```diff
@@ -0,0 +1,28 @@
+spec:
```

Comment:
dont think we need another folder here. we can prob have tools/monitoring/sampleConfig.yaml

Reply:
also, missing header